muxstatssdkbitmovinplayer/src/main/java/com/mux/stats/sdk/muxstats/bitmovinplayer/MuxStatsSDKBitmovinPlayer.java

```diff
@@ -519,12 +519,12 @@ public Float getSourceAdvertisedFramerate() {
 
     @Override
     public Long getSourceDuration() {
-        return secondsToMs(player.get().getPlayer().getDuration());
+        return secondsToMs(playbackPosition);
     }
 
     @Override
     public boolean isPaused() {
-        return player.get().getPlayer().isPaused();
+        return state == PlayerState.PAUSED;
```

Comment:
Are we sure that this change is safe?

For example, when paused while seeking would the previous implementation have returned true or false? (From what I can tell the new one would return false).